protobuf-test/build.rs

```diff
@@ -16,17 +16,15 @@ use protobuf_test_common::build::*;
 fn gen_in_dir(dir: &str, include_dir: &str) {
     gen_in_dir_impl(
         dir,
-        include_dir,
         |GenInDirArgs {
              out_dir,
              input,
-             includes,
              customize,
          }| {
             protoc_rust::Codegen::new()
                 .out_dir(out_dir)
                 .inputs(input)
-                .includes(includes)
+                .includes(&["../proto", include_dir])
```

Comment:
Can we make it work without this line? For compatibility with `protoc` which does not require it?

Reply:
So this is actually just inlining a slice that used to live here: https://github.com/stepancheg/rust-protobuf/pull/507/files#diff-d9930b6eababb528edc6d13b26e6d004L223

In other words, protobuf-test has always needed to specify the "../proto" include directory in order to pick up rustproto.proto. (It *is* true that you don't need to specify this if you just want the "google/protobuf" imports, but the protobuf-test photos expect to be able to import "rustproto.proto". I think ideally somehow rustproto.proto would be shipped with `protoc_rust`, but I'm not sure about how to rig that up.)

The change in this PR is to allow protobuf-codegen-pure-test to avoid specifying "../proto", since it's no longer required.

Reply:
Ah, it was there before, but specified in common.

I see.